sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -314,3 +315,16 @@ def test_default_labels(pyplot, display_labels, expected_labels):
 
     assert_array_equal(x_ticks, expected_labels)
     assert_array_equal(y_ticks, expected_labels)
+
+
+def test_error_on_a_dataset_with_unseen_labels(pyplot, fitted_clf, data):
+    X, y = data
+
+    # get values that are not present in 'fitted_clf.classes_'
+    y = y + 1
+    disp = plot_confusion_matrix(fitted_clf, X, y,
+                                 labels=None, display_labels=None)
```

Comment:
```suggestion
    disp = plot_confusion_matrix(fitted_clf, X, y)
```

Reply:
Thank you very much for your attention and help!
I wanted to make sure that the _plot_confusion_matrix_() function is called with _labels=None_ and _display_labels=None_ even if the default behaviour is changed in future. Should I still remove these arguments?

Reply:
In this case, we want to test the normal use case, where a user does not provided the defaults.

If the defaults change, we would update this test.